api/src/main/java/org/openmrs/util/ConfigUtil.java

```diff
@@ -0,0 +1,113 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.util;
+
+import org.apache.commons.lang3.StringUtils;
+import org.openmrs.GlobalProperty;
+import org.openmrs.api.GlobalPropertyListener;
+import org.openmrs.api.context.Context;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * A utility class for working with configuration properties
+ */
+public class ConfigUtil implements GlobalPropertyListener {
+
+	/**
+	 * Cache of global property key/value pairs to enable lookups that do not require accessing the service each time
+	 */
+	private static final Map<String, String> globalPropertyCache = new HashMap<>();
+	
+	/**
+	 * Gets the value of the given OpenMRS global property
+	 */
+	public static String getGlobalProperty(String propertyName) {
+		if (globalPropertyCache.containsKey(propertyName)) {
+			return globalPropertyCache.get(propertyName);
+		}
+		String value = Context.getAdministrationService().getGlobalProperty(propertyName);
+		globalPropertyCache.put(propertyName, value);
+		return value;
+	}
+
+    /**
+	 * Returns the value of the given OpenMRS runtime property
+	 */
+	public static String getRuntimeProperty(String propertyName) {
+		return Context.getRuntimeProperties().getProperty(propertyName);
+	}
+
+	/**
+	 * Returns the value of the given OpenMRS runtime property
+	 */
+	public static boolean hasRuntimeProperty(String propertyName) {
+		return Context.getRuntimeProperties().containsKey(propertyName);
+	}
+
+	/**
+	 * Returns the value of the given OpenMRS system property
+	 */
+	public static String getSystemProperty(String propertyName) {
+		return System.getProperty(propertyName);
+	}
+
+	/**
+	 * Returns the value of the given OpenMRS runtime property
```

Comment:
You probably forgot to change the above description.

Reply:
Good point, fixed